doc/gen/src/main/scala/GenDocs.scala

```diff
@@ -57,40 +84,101 @@ object GenDocs extends App {
       }
     }.toMap
 
-    val table = List("Code", "Mnemonic", "Description") :: {
+    val view = table("Code", "Mnemonic", "Description") {
       Operation.mnemonicByOpcode.toList.sortBy(_._1) map {
         case (opcode, mnemonic) =>
           List("0x%02X".format(opcode), mnemonic, descriptionByOpcde.getOrElse(opcode, ""))
       }
     }
 
-    // measure columns
-    val sizes = table.foldLeft(List.fill(table.head.length)(0)) {
-      case (acc, xs) =>
-        acc.zip(xs).map {
-          case (l, r) => Math.max(l, r.length)
+    new PrintWriter("doc/ref/vm/opcodes.md") {
+      write(DoNotEditWarn)
+      write("# Pravda VM opcodes\n")
+      write(view)
+      close()
+    }
+  }
+
+  def genStdlibDocs() = {
+
+    def nameToFileName(x: String) = {
+      x.charAt(0).toLower + x.toList.tail
+        .flatMap {
+          case s if s.isUpper => List('-', s.toLower)
+          case s              => List(s)
         }
+        .mkString
+        .stripSuffix("$")
     }
 
-    val header :: rows = table map { cols =>
-      cols
-        .zip(sizes)
-        .map { case (col, size) => col + (" " * (size - col.length)) }
-        .mkString("|")
+    val docs = StandardLibrary.All.map { f =>
+      val fileName = nameToFileName(f.name)
+
+      def typeToName(x: Data.Type) = x match {
+        case Data.Type.Bytes   => "bytes"
+        case Data.Type.Number  => "number"
+        case Data.Type.Null    => "null"
+        case Data.Type.Boolean => "boolean"
```

Comment:
It would be nice to have one name for boolean in bytecode mnemonic and in asm. It is "bool" in asm.